locales/de-DE.json

```diff
@@ -276,17 +452,21 @@
     "year_past": "vor 0 Jahren|letztes Jahren|vor {n} Jahren"
   },
   "timeline": {
-    "show_new_items": "Zeige {v} neue Beiträge|Zeige {v} neuen Beitrag|Zeige {v} neue Beiträge"
+    "show_new_items": "Zeige {v} neue Beiträge|Zeige {v} neuen Beitrag|Zeige {v} neue Beiträge",
+    "view_older_posts": "Ältere Beiträge aus anderen Instanzen werden möglicherweise nicht angezeigt."
   },
   "title": {
     "federated_timeline": "Föderierte Timeline",
     "local_timeline": "Lokale Timeline"
   },
   "tooltip": {
     "add_content_warning": "Inhaltswarnung hinzufügen",
+    "add_emojis": "Emojis hinzufügen",
     "add_media": "Bilder, ein Video oder eine Audiodatei hinzufügen",
+    "add_publishable_content": "Füge Inhalte zum Veröffentlichen hinzu",
     "change_content_visibility": "Sichtbarkeit von Inhalten ändern",
     "change_language": "Sprache ändern",
+    "emoji": "Emoticons",
```

Comment:
we should keep this translation consistent, in "add_emojis" it's translated as "emoji", I'd also prefer it like that here

Reply:
```suggestion
    "emoji": "Emoji",
```

Reply:
Iirc is "Emojos" a more common phrase in Mastodon.

Reply:
> Iirc is "Emojos" a more common phrase in Mastodon.

Never heard of `Emojos`, is it a typo?

Reply:
I saw it mentioned in https://emojos.in

(“emojo” is the semi-jokey term for custom emoji on [Mastodon](https://joinmastodon.org/).)